### PR TITLE
Properly fetch tag refs

### PR DESCRIPTION
**Pull request by geigerzaehler**
We now include tags properly in the list of refs to be fetched from a peer.

### CODE REVIEW

File: librad/src/git/types/reference.rs [hunk 1]
```diff
@@ -261,6 +261,18 @@ impl<N, R> Reference<N, R, One> {
             namespace: namespace.into(),
         }
     }
+
+    /// Build a reference that points to:
+    /// * `refs/namespaces/<namespace>/refs/tag/<name>`
+    /// * `refs/namespaces/<namespace>/refs/remote/<peer_id>/tag/<name>
```
Comment (kim) [inline librad/src/git/types/reference.rs:267]:
```suggestion
    /// * `refs/namespaces/<namespace>/refs/tags/<name>`
    /// * `refs/namespaces/<namespace>/refs/remote/<peer_id>/tags/<name>
```

**Review by kim — APPROVED** [review]
Looks good, thanks!

As these mappings are a frequent source of confusion, would be great to amend the commit message with some hints as to what was wrong before, and is now fixed.

**Review by FintanH — COMMENTED** [review]
Looks good :+1: Just some typos :innocent:

**Comment by geigerzaehler** [comment]
@kim, @FintanH. Could you have another look? I’ve fixed the typos and merged master into the PR.

**Review by FintanH — APPROVED** [review]
Looks like it needs a merge of master :)

**Comment by geigerzaehler** [comment]
Merged in b6f3ffcdfd7f4ade57f6c1aab2032faa96f808c8

**Pull request closed without merging**